src/services/chats.js

```diff
@@ -1,7 +1,140 @@
 const { chatModel, userModel } = require("../modules/stores/mongo");
+const { chatPopulateOption } = require("../modules/populates/chats");
 const awsS3 = require("../modules/stores/awsS3");
+const { transformChatsForRoom, emitChatEvent } = require("../modules/socket");
 
-const { emitChatEvent } = require("./socket.chats");
+const loadRecentChatHandler = async (req, res) => {
+  try {
+    const io = req.app.get("io");
+    const roomId = req.body.roomId;
+    const userId = req.session?.userId;
+    const user = await userModel.findOne({ id: userId });
+    if (!userId) {
+      return res.status(403).send("Chat/ : socket did not connected");
+    }
```

Comment:
socket 연결 여부는 

```suggestion
    const { socketId } = req.session;
    if (!socketId || !io) {
      return res.status(403).send("Chat/ : socket did not connected");
    }
```

로 확인해야 할 것 같아요